### PR TITLE
Implement Redis for session storage

### PR DESCRIPTION
**Pull request by trev-dev**
Jira: OFMCC-4121

### CODE REVIEW

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: backend/src/util/redis/redis-client.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
-'use strict';
```
[the remark this answers is not in the document]
Comment (trev-dev) [reply backend/src/util/redis/redis-client.js:1]: I've been doing a lot of work refactoring CJS to ESM on another project, deleting these has become a tick.  Reverting.

File: backend/src/config/index.js [hunk 2]
```diff
@@ -59,12 +59,10 @@ nconf.defaults({
     natsCluster: process.env.NATS_CLUSTER,
   },
   redis: {
-    use: process.env.USE_REDIS,
+    use: process.env.REDIS_ENABLE,
```
[the remark this answers is not in the document]
Comment (trev-dev) [reply backend/src/config/index.js:62]: I did :)

File: backend/src/app.js [hunk 3]
```diff
@@ -72,7 +72,8 @@ const logStream = {
   },
 }
 
-const dbSession = getRedisDbSession()
+const dbSession = config.get('redis:enable') == 'true' ? getRedisDbSession(session) : undefined
```
[the remark this answers is not in the document]
Comment (trev-dev) [reply backend/src/app.js:75]: Nice catch with the triple equals, too.

File: backend/src/app.js [hunk 4]
```diff
@@ -39,8 +39,8 @@ const paymentsRouter = require('./routes/payments')
 const reportsRouter = require('./routes/reports')
 const { MappableObjectForBack } = require('./util/mapping/MappableObject')
 const { RoleMappings } = require('./util/mapping/Mappings')
+const { getRedisDbSession } = require('./util/redis/redis-client');
```
[the remark this answers is not in the document]
Comment (trev-dev) [reply backend/src/app.js:42]: Bad pinky.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**